### DISCUSSION TITLE
emacs depends on unbound?

### DISCUSSION THREAD

**Discussion by rexbinary** [Homebrew/discussions#3693] [Everyday usage] [answered]
Hi all,

Does anyone know why installing emacs would install the DNS resolver unbound? This has to be a bug I suppose?

`❯ brew uninstall unbound
Error: Refusing to uninstall /usr/local/Cellar/unbound/1.16.3
because it is required by emacs and gnutls, which are currently installed.
You can override this and force removal with:
  brew uninstall --ignore-dependencies unbound`

**Comment by carlocab** [comment, marked as answer]
`emacs` depends on `gnutls`, and `gnutls` depends on `unbound`.

**Reply by rexbinary** [reply]
Thanks. What a bad design choice.
